media/js/src/exerciseSpace.tsx

```diff
@@ -209,6 +216,7 @@ export const ExerciseSpace: React.FC = () => {
         });
         {void fetchSolutions();}
         window.scrollTo(0, 0);
+        ReactGA.pageview(window.location.pathname + window.location.search);
```

Comment:
I'm seeing a mismatch in the page event being sent when the user navigates from the question space to the exercise space.

```
location: http://localhost/questions/2
page: /exercise/7
```

A refresh on the exercise space page shows the correct location/page:
```
location: http://localhost/exercise/7
page: /exercise/7
```

(Not sure if this is exactly where the problem is btw. I'm using the Google Analytics Debugger in Chrome to introspect the events and page hits.)

Reply:
Hmmmm....looking at this again. Is the "location" the "referrer"?

Reply:
Using the debugger, I've verified that the page is being tracked correctly. Not sure what's up with the location field, but it does not seem to matter with GA itself. You can ignore this comment.